pkgs/applications/networking/cluster/terraform-providers/default.nix

```
@@ -17,15 +17,18 @@ let
     ({ owner
      , repo
      , rev
-     , version
+     , spdx ? "UNSET"
+     , version ? "${lib.removePrefix "v" rev}"
```

```suggestion
     , version ? lib.removePrefix "v" rev
```
the interpolation is not needed here